smdebug_rulesconfig/actions/actions.py

```diff
@@ -168,7 +168,7 @@ def __init__(self, phone_number: str):
         }
         ```
 
-        :param phone_number: Phone number to send the SMS to.
+        :param phone_number: 10 digit phone number preceded by a `+`.
```

Comment:
is +10 digit too strict? can it be something else out of United States ?

Reply:
good catch, I'll update this so that it can accept a number from 10-15 digits (can a valid phone number be longer or shorter?)

Reply:
Done.